tests/test_file_ops.py

```diff
@@ -0,0 +1,66 @@
+import shutil
+import os
+from pathlib import Path
+import sys
+sys.path.insert(1, "nesta_ds_utils/")
+import file_ops
+import pytest
+
+
+@pytest.fixture
+def test_output_path():
+    """generates pathlib.Path to dump intermediate test data
+
+    Yields:
+        pathlib.Path: output path to dump intermetiate test data
+    """
+    test_output_path = file_ops._convert_str_to_pathlib_path("tests/temp/")
+    file_ops.make_path_if_not_exist(test_output_path)
+    yield test_output_path
+    shutil.rmtree("tests/temp/")
+
+
+def test_convert_str_to_pathlib_path(test_output_path):
+    """tests that file_ops method convert_str_to_pathlib_path returns type patlib.Path
```

Comment:
```suggestion
    """Tests that file_ops method convert_str_to_pathlib_path returns type patlib.Path.
```
Is there a way to render a different styling for functions, variables, types etc. in sphinx with markup in the docstring?